File: programme/area-of-circle/AreaOfCircle.cs
// Write a programme to find the area of circle
using System;

public class AreaOfCircle
{
    public static void Main(String[] args)
    {
        Console.Write("Enter the radius : ");
        int r = int.Parse(Console.ReadLine());
        double area = Math.PI * r * r;
        double answer = Math.Round(area, 2);

        Console.WriteLine("Area of circle   : " + answer);
    }
}

File: programme/area-of-circle/area-of-circle.js
// Write a programme to find the area of circle
const areaOfCircle = function areaOfCircle(r) {
	return Math.PI * r * r;
};

const readline = require("readline");
const rl = readline.createInterface({
	input: process.stdin,
	output: process.stdout,
});

rl.question("Enter radius : ", (userInput) => {
	const r = Number.parseFloat(userInput);
	if (!isNaN(r)) {
		area = areaOfCircle(r);
		console.log("\nArea : ", area.toPrecision(4));
		rl.close();
	} else {
		console.log("Invalid input: the input is not a number.");
		rl.close();
	}
});
